src/components/CategorySelect/components/SuggestionsList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Header, ListBoxItem } from 'react-aria-components'
+import { Text, TextSize } from '../../Typography'
+import { CategoryOption } from '../types'
+import { VStack } from '../../ui/Stack/Stack'
+import { findParentCategory, isSelected } from '../utils'
+import { Category } from '../../../types'
+import CheckIcon from '../../../icons/Check'
+import { Tooltip, TooltipContent, TooltipTrigger } from '../../Tooltip/Tooltip'
+import InfoIcon from '../../../icons/InfoIcon'
+import { ListSection } from './ListSection'
+
+type SuggestionsListProps = {
+  suggestions?: CategoryOption[]
+  categories?: Category[]
+  selected?: CategoryOption
+}
+
+export const SuggestionsList = ({ suggestions, categories, selected }: SuggestionsListProps) => {
+  if (!categories || !suggestions || suggestions.length === 0) {
+    return
+  }
+
+  return (
+    <ListSection aria-label='Suggestions'>
+      <Header slot='header'>
+        <Text size={TextSize.xs} status='disabled'>Suggestions</Text>
+      </Header>
+      {suggestions.map((option, index) => (
+        <ListBoxItem
+          key={index}
+          id={`suggestion-${option.payload.id}`}
```

Comment:
The `suggestion-` prefix resolves strange ComboBox behavior. Without prefix, all categories are rendered in `Suggestions` section:

![image](https://github.com/user-attachments/assets/e3f946da-c5b7-43fe-bfab-97a0a80fb79c)


The expected result:

![image](https://github.com/user-attachments/assets/d1031adc-9e29-4f27-b366-d5e14fd674e5)

Probably this happens due to duplicated id - another item in "All categories" has same category id